/*
Copyright 2020 The Magma Authors.

This source code is licensed under the BSD-style license found in the
LICENSE file in the root directory of this source tree.

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
*/

syntax = "proto3";

package magma.lte;

option go_package = "magma/lte/cloud/go/protos";

enum EnbOffloadType {
    ALL = 0;
    ANY = 1;
    ANY_CONNECTED = 2;
    ANY_IDLE = 3;
}

message StartAgwOffloadRequest {
    uint32 enb_id = 1;
    EnbOffloadType enb_offload_type = 2;
    string imsi = 3;
}

message StartAgwOffloadResponse {}

service HaService {
    rpc StartAgwOffload (StartAgwOffloadRequest) returns (StartAgwOffloadResponse) {}
}
